load-testing/config/load_test_manifest_reader.go

```diff
@@ -11,15 +11,23 @@ import (
 type ProvisionedActorConfig struct {
 	// KeyName is the **name** of the key in the keyring to be used by the given actor.
 	KeyName string `yaml:"key_name"`
+	// Address is the address of the actor, which is used to identify already staked
+	// actors in the network in persistent chains.
+	Address string `yaml:"address"`
```

Comment:
Why can't we use `keyName` to determine the the address?

1. We already have helpers for this.
2. I feel like this creates two sources of truth.
3. I feel like we're getting into a weird scenario where there's implicit institutional knowledge

Reply:
We can't use the `keyName` in that case because the `keyName->address` association comes from the fact that we have that key present the test environment's `keyRing`. Which is not the case for TestNet where `Gateways` are owned and operated by other's and we don't have their keys in our kyring.

I agress that there's no explicit indications on whether to provide `address`es or `keyNames` and feel like comments are not enough.

An alternative would be to have different manifests for each case. It still remains that the `is_ephemeral` flag is the one that dictates whether the test targets a `LocalNet` or a `TestNet`. Rather than the filename.

We may also have dedicated manifest parsers for clarity too.

Reply:
Can you #PUC with a a short TODO_IMPROVE, and link to this thread (https://github.com/pokt-network/poktroll/pull/540/files#r1607291985) as well?